src/main/java/com/develhope/spring/vehicle/models/Vehicle.java

```diff
@@ -35,7 +35,7 @@ public String getEngine() {
     }
 
     public BigDecimal getDiscountedPrice() {
-        return discountedPrice;
+        return discountedPrice.setScale(2, RoundingMode.HALF_EVEN);
```

Comment:
in realtà nei get non è necessario penso specificarlo poichè ti dovrebbe ritornare in questo caso il discountedPrice già con il setScale preso dal costruttore, però non ne sono certo e quindi lascio giusto il commento e chiediamo a Lorenzo

Reply:
Era lo stesso dilemma mio, ma credo che ribadirlo prevenga qualsiasi errore. Magari chiediamo a Lorenzo oggi (tra poco)

Reply:
ora lo cambio e lo tolgo anche dagli altri getter